lucene/translate.py

```diff
@@ -154,36 +169,43 @@ def postprocess_translated_objc(path):
             if any(fnmatch.fnmatch(full_path_java, ptn) for ptn in excluded):
                 continue
 
-            full_path_m = full_path_java.replace(
-                src, dst).replace(".java", ".m")
+            full_path_m = full_path_java.replace(src, dst).replace(".java", ".m")
             if os.path.exists(full_path_m):
                 if os.path.getmtime(full_path_m) >= os.path.getmtime(full_path_java):  # nopep8
                     continue
+
             to_compile.append(full_path_java)
             to_postprocess.append(full_path_m)
 
-    print('%s => %s files to compile' % (src, len(to_compile)))
+    print('\nTranslating: %s with %d java files to compile.\n' % (src, len(to_compile)))
 
     if to_compile:
-        print('Compiling %d java files' % len(to_compile))
         args = [
-            'j2objc',
-            # '-use-arc',
-            '-classpath', classpaths,
-            '--segmented-headers',
-            '-sourcepath', src,
+            j2objc,
             '-d', dst,
+            '-classpath', ':'.join(classpaths),
+            '--swift-friendly',
```

Comment:
Hi,

In iOS development Swift is kinda always used for new projects, obj-c is used only for the old ones that have been developed before Swift. 

The flags `--swift-friendly` and `--nullability` add many getter that help when using the code in Swift. I guess it should also add the `_Nonnull` and `_Nullable` keywords helping Swift to know wether a variable is an optional or not.
Right now they add some getters like this one:
```objc
+ (OrgApacheLuceneAnalysisAnalyzer_ReuseStrategy *)GLOBAL_REUSE_STRATEGY {
    return OrgApacheLuceneAnalysisAnalyzer_GLOBAL_REUSE_STRATEGY;
}
```


About `--no-extract-unsequenced`, the documentation says:
```
Don't rewrite expressions that would produce unsequenced modification errors.
```
I thought it would have been helpful, but I am not sure "how great" it is. This is what it does:
```
-    jint unseq$1 = self->jjnewStateCnt_;
-    if ((i = unseq$1) == (startsAt = 3 - (self->jjnewStateCnt_ = startsAt))) return curPos;
+    if ((i = self->jjnewStateCnt_) == (startsAt = 3 - (self->jjnewStateCnt_ = startsAt))) return curPos;
```

I did not change anything there as I think they are useful.

Tell me what you think 😃